app/views/static_pages/landing.html.slim

```diff
@@ -1 +1,6 @@
 h1= @greeting
+
+
+
+= link_to 'Page sample'
+
```

Comment:
This link is useless here. We had moved such stuff to navigation bar. I think it appeared here again as the result of rebase. Please, remove it